CHANGELOG.md

```diff
@@ -9,33 +84,33 @@ _Jun 7, 2022_
 A big thanks to the 15 contributors who made this release possible.
 This release is mostly about 🐛 bug fixes and 📚 documentation improvements.
 
-## `@mui/material@5.8.3`
+### `@mui/material@5.8.3`
 
 - [Alert] Constrain message width and allow overflow (#32747) @Janpot
 - [Checkbox] Add support for CSS variables (#32579) @haneenmahd
 - [Slider] Fix positioning of tooltips on vertical slider (#32919) @abhinav-22-tech
 
-## `@mui/system@5.8.3`
+### `@mui/system@5.8.3`
 
 - [system] Configurable attributes for libraries (#32971) @siriwatknp
 
-## `@mui/codemod@5.8.3
+### `@mui/codemod@5.8.3
```

Comment:
thanks for correcting this :)
I've noticed I also missed a closing tick here - could you please add it?

```suggestion
### `@mui/codemod@5.8.3`
```